frontend/src/metabase/containers/QuestionLoader.jsx

```diff
@@ -71,7 +71,7 @@ const QuestionLoader = ({
       children={children}
     />
   ) : // if there's a questionHash it means we're in ad-hoc land
-  questionHash != null ? (
+  questionHash != null && questionHash !== "" ? (
```

Comment:
The use of this component on `/_internal/question` was broken since `questionHash` was passed as an empty strnig.